packages/sdk/src/index.ts

```diff
@@ -109,6 +130,63 @@ async function scheduleXcmpTaskWithPayThroughRemoteDerivativeAccountFlow({
   return sendExtrinsicResult;
 }
 
+async function scheduleXcmpPriceTaskWithPayThroughRemoteDerivativeAccountFlow({
+  oakAdapter,
+  destinationChainAdapter,
+  taskPayloadExtrinsic,
+  scheduleFeeLocation,
+  executionFeeLocation,
+  automationPriceTriggerParams,
+  scheduleAs,
+  keyringPair,
+}: ScheduleXcmpPriceTaskWithPayThroughRemoteDerivativeAccountFlowParams): Promise<SendExtrinsicResult> {
+  const oakApi = oakAdapter.getApi();
+  const { paraId, xcmInstructionNetworkType, xcm } = destinationChainAdapter.getChainData();
+  if (_.isUndefined(paraId)) throw new Error("chainData.paraId not set");
+  if (_.isUndefined(xcmInstructionNetworkType)) throw new Error("chainData.xcmInstructionNetworkType not set");
+  if (_.isUndefined(xcm)) throw new Error("chainData.xcm not set");
```

Comment:
Please do this
```
if (_.isUndefined(xcm))
{
    throw new Error("chainData.xcm not set");
}
```

instead of the one-line style.

Does `eslint --fix` accept this one-line style?

Reply:
Fixed

Reply:
Actually, if eslint is not complaining about the style. We don’t need to change it. We will just follow formats defined in eslint.

I ran `npm run lint` across all files.
https://github.com/OAK-Foundation/oak.js/commit/af257ea8b264e12d13427663757fe9a714816806

Reply:
Also all changed file in a commit will run `eslint` by `husky` now.
https://github.com/OAK-Foundation/oak.js/commit/db57d573ed02eaf3fd484d4c03fdd16051f15e68